.gitignore

```diff
@@ -1,4 +1,5 @@
 .env
+env
```

Comment:
why new file?

Reply:
Just added `env` to ignore virtual environment . It's common across .gitignore